cs/composer.texy

```diff
@@ -121,7 +121,7 @@ Náš základní `composer.json` tedy může vypadat takto
 }
 \--
 
-Říkáme zde, že naše aplikace (nebo knihovna) vyžaduje balíček `nette/nette` a chce nejnovější verzi, která odpovídá `~2.4.0` (tj. nejnovější setinkovou verzi Nette 2.4), a taky že běží pouze na PHP `5.6.1` a novějším.
+Říkáme zde, že naše aplikace (nebo knihovna) vyžaduje balíček `nette/nette` a chce nejnovější verzi, která odpovídá `~2.4.0` (tj. nejnovější setinkovou verzi Nette 2.4), a taky že běží pouze na PHP `5.6.1` a novějším. Doporučujeme však používat nejnovější PHP 7.2.
```

Comment:
Tu větu bych tam asi nedával, když někomu vysvětluješ composer, je divný ho tlačit k updatu PHP.